server/service/osquery_utils/queries.go

```diff
@@ -2058,6 +2059,130 @@ func directIngestMDMDeviceIDWindows(ctx context.Context, logger log.Logger, host
 	return ds.UpdateMDMWindowsEnrollmentsHostUUID(ctx, host.UUID, rows[0]["data"])
 }
 
+var luksVerifyQuery = DetailQuery{
```

Comment:
We will need a discovery query (`DetailQuery.Discovery`) for vanilla osquery hosts (to check if the `lsblk` and `cryptsetup_luks_salt ` exist), or check `host.OrbitNodeKey != nil` in `IsLUKSSupported`. Probably better to do both (to also support customers with old orbit versions)